### PR TITLE
[JBPM-10043] Test for EventEmitter status after delegate

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**: [JBPM-10043](https://issues.redhat.com/browse/JBPM-10043)

### CODE REVIEW

**Review by afalhambra — APPROVED** [review]
nice one @gmunozfe!

File: kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-kafka-sample/src/test/java/org/kie/server/springboot/samples/KafkaEmitterHappyPathTest.java [hunk 1]
```diff
@@ -214,5 +214,25 @@ public void testEmitterRecordTooLargeException() {
...
+        assertNotNull(processInstanceId);
+        assertTrue(processInstanceId > 0);
+        
+        List<TaskSummary> tasks = runtimeDataService.getTasksAssignedAsPotentialOwner(JOHN, new QueryFilter());
+        assertEquals(1, tasks.size());
+        
+        // delegating task to user Yoda (Reserved status)
+        userTaskService.delegate(deploymentId, tasks.get(0).getId(), JOHN, YODA);
+        
+        consumeAndAssertTaskRecords(TASKS_TOPIC, "Reserved", 2);
```
[the remark this answers is not in the document]
Comment (gmunozfe) [reply kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-kafka-sample/src/test/java/org/kie/server/springboot/samples/KafkaEmitterHappyPathTest.java]: yes, there are 2 events, one before delegation (actualOwner=john) and another after it (actualOwner=yoda). I've added this assertion according to your suggestion. Good point @afalhambra !

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
@mareknovotny this is ready to merge

**Pull request merged**